classes/journal/JournalDAO.inc.php

```diff
@@ -204,7 +204,7 @@ function deleteJournalById($journalId) {
 	 * @param $enabledOnly boolean True iff only enabled jourals wanted
 	 * @param $rangeInfo object optional
 	 * @param $sortBy JOURNAL_FIELD_... optional sorting parameter
-	 * @param $searchField JOURNAL_FIELD_... optional filter parameter
+	 * @param $searchField JOURNAL_FIELD_... optional filter parameter; DEPRECATED - use getBySetting() instead
```

Comment:
The `getBySetting` function will deal with entries in `journal_settings`, but not `JOURNAL_FIELD_SEQUENCE`, which is also supported here. I'd suggest benign neglect (just leaving it as is). The entire function has already been removed from the master branch, I think.